includes/class-llms-form-templates.php

```diff
@@ -0,0 +1,680 @@
+				'label'          => __( 'Confirm Email Address', 'lifterlms' ),
+				'match'          => 'email_address',
+				'data_store_key' => false,
+			)
+		);
+
+		return $this->wrap_columns( $cols, 'logged_out' );
+
+	}
+
+	/**
+	 * Retrieve blocks for the name row.
+	 *
+	 * Returns an empty array if names are disabled by legacy settings.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $location Form location. Accepts template options passed to `$this->get_template()`.
+	 * @return array
+	 */
+	protected function get_row_names( $location ) {
+
+		$cols = array();
+
+		$option = get_option( sprintf( 'lifterlms_user_info_field_names_%s_visibility', $location ), 'required' );
+		if ( 'hidden' === $option ) {
+			return $cols;
+		}
+
+		$cols[] = $this->get_block(
+			'user-first-name',
+			array(
+				'id'       => 'first_name',
+				'label'    => __( 'First Name', 'lifterlms' ),
+				'required' => ( 'required' === $option ),
+			)
+		);
+
+		$cols[] = $this->get_block(
+			'user-last-name',
+			array(
+				'id'       => 'last_name',
+				'label'    => __( 'Last Name', 'lifterlms' ),
+				'required' => ( 'required' === $option ),
+			)
+		);
+
+		return $this->wrap_columns( $cols );
+
+	}
+
+	/**
+	 * Retrieve blocks for the password row.
+	 *
+	 * @since [version]
+	 *
+	 * @return array
+	 */
+	protected function get_row_password() {
+
+		$cols = array();
+
+		$cols[] = $this->get_block(
+			'user-password',
+			array(
+				'field' => 'password',
+				'id'    => 'password',
+				'label' => __( 'Password', 'lifterlms' ),
+				'match' => 'password_confirm',
+			)
+		);
+
+		$cols[] = $this->get_block(
+			'user-password-confirm',
+			array(
+				'field'          => 'password',
+				'id'             => 'password_confirm',
+				'label'          => __( 'Confirm Password', 'lifterlms' ),
+				'match'          => 'password',
+				'data_store_key' => false,
+			)
+		);
+
+		return $this->wrap_columns( $cols, 'logged_out' );
+
+	}
+
+	/**
+	 * Retrieve the password strength meter field row.
+	 *
+	 * @since [version]
+	 *
+	 * @return array
+	 */
+	protected function get_row_password_meter() {
+
+		$option = llms_parse_bool( get_option( 'lifterlms_registration_password_strength' ) );
+		if ( ! $option ) {
+			return array();
+		}
+
+		return array(
+			$this->get_block(
+				'password-strength-meter',
+				array(
+					'className'       => 'llms-password-strength-meter',
+					'field'           => 'html',
+					'id'              => 'llms-password-strength-meter',
+					'required'        => false,
+					'description'     => sprintf( __( 'A %1$s password is required. The password must be at least %2$s characters in length. Consider adding letters, numbers, and symbols to increase the password strength.', 'lifterlms' ), '{min_strength}', '{min_length}' ),
+					'min_strength'    => get_option( 'lifterlms_registration_password_min_strength', 'strong' ), // Use legacy option.
+					'min_length'      => 6,
+					'llms_visibility' => 'logged_out',
```

Comment:
Same as above, this used in the dashboard edit account form as well, so it should be visible to all... right?